build_qcow2.sh

```diff
@@ -20,6 +20,10 @@ docker volume rm build_debian_iso_ext 2>/dev/null
 
 set -ex
 
```

Comment:
same, build_tmp must be created

Reply:
I will use .gitkeep files to have the folder created when cloning the repo.